src/main/java/com/factly/dega/service/impl/OrganizationServiceImpl.java

```diff
@@ -152,4 +160,40 @@ public void delete(String id) {
             .map(organizationMapper::toDto);
     }
 
+    /**
+     * Get the organizations by keycloakId and pageable.
+     *
+     * @param keycloakId the keycloakId of the User
+     * @param pageable the pagination information
+     *
+     * @return List<OrganizationDTO> List of Organizations by keycloakId and pageable
+     */
+    public List<OrganizationDTO> getOrganizations(String keycloakId, Pageable pageable){
+
+        Optional<DegaUserDTO> degaUserDTO = degaUserService.findByKeycloakId(keycloakId);
```

Comment:
if user keycloakId is null then we should return all organizations just like before without this additional param

Reply:
done